apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/BaseNode/BaseNode.tsx

```diff
@@ -20,6 +20,13 @@ import {
 import { useEditor } from '@core/journeys/ui/EditorProvider'
 import ArrowRightIcon from '@core/shared/ui/icons/ArrowRight'
 
+export enum HandleVariant {
+  NONE = 'none', // not rendered - Default value
+  SHOWN = 'shown', // rendered, visible and connectable
+  HIDDEN = 'hidden', // rendered, not visible or connectable.
+  DISABLED = 'disabled' // rendered, visible, not connectable
```

Comment:
Just to follow the pattern already set by other enums
```suggestion
  None = 'none', // not rendered - Default value
  Shown = 'shown', // rendered, visible and connectable
  Hidden = 'hidden', // rendered, not visible or connectable.
  Disabled = 'disabled' // rendered, visible, not connectable
```